app/views/articles/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<%= link_to_unless_current "English", locale: 'en' %>
```

Comment:
Is it only on the main page the user can switch between languages? if not move it to the application.html.erb file

Reply:
done